queue.h

```diff
@@ -0,0 +1,48 @@
+#ifndef CONCURRENT_QUEUE_H
+#define CONCURRENT_QUEUE_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#ifdef _WIN32
+#if defined(CQ_STATIC) //preferred way
+#define CQ_EXPORT
+#elif defined(DLL_EXPORT)
+#define CQ_EXPORT __declspec(dllexport)
+#else
+#define CQ_EXPORT __declspec(dllimport)
+#endif
+#else
+/*
+need non-windows version here
```

Comment:
Please add `#error` instead of leaving `CQ_EXPORT` undefined which will explode later in unrelated parts of the code.

Reply:
should it leave a message saying to contact the maintainer and have your email address or something?

Reply:
no, just that `*nix` support is untested